### PR TITLE
Added the source tracking extenstion and Feature to track origin source

### PR DESCRIPTION
**Pull request by sameer-bot1**
Goal:- Our objective is to determine the origin source of each registration and, upon form submission, store this origin source along with the contact details.

Solution:-

To achieve this here are all the CiviCRM dashboard changes:- 
1. Added source tracker extension and installed it from CiviCRM extension 
2. Added custom field group  named as `Source Tracking`
3. Added custom field named as  `Source`
4. Added Source field to `Name and Address` profile
5. Used this profile to Registration form

### CODE REVIEW

File: civicrm-2024-06-06-73231bd.sql
Comment (pokhiii) [inline civicrm-2024-06-06-73231bd.sql:1]: How is this file generated? It needs to be removed.

File: wp-content/upgrade/civicrm-config-log/ConfigAndLog/.htaccess
Comment (pokhiii) [inline wp-content/upgrade/civicrm-config-log/ConfigAndLog/.htaccess:1]: This file is not needed can you remove it and add `wp-content/upgrade/` to `.gitignore`

**Review by pokhiii — CHANGES_REQUESTED** [review]

File: civicrm-2024-06-06-73231bd.sql
Comment (sameer-bot1) [reply civicrm-2024-06-06-73231bd.sql:1]: ok sure

**Comment by pokhiii** [comment]
Closing this as I have implemented this separately.

**Pull request closed without merging**